static/examples/5.x/conditional-screens.js

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Text, View, Button } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+const Stack = createNativeStackNavigator();
```

Comment:
`@react-navigation/native-stack` didn't exist yet in v5. Use `@react-navigation/stack` in v5 examples but encourage to use `@react-navigation/native-stack` in v6 and v7 wherever possible